cypress/pages/AddEmployeePage.ts

```diff
@@ -0,0 +1,34 @@
+import BasePage from "./BasePage";
+
+class AddEmployeePage extends BasePage {
+    // locators
+    firstNameInput: string = "[name='firstName']";
```

Comment:
we should not def the type since we assign the value
typescript will take care of the type in this case